app/models/submissions/questions_sub.rb

```diff
@@ -4,6 +4,22 @@
 class QuestionsSub < Submission
   include SubmissionsHelper
   validate :check_time_taken
+
+
+  def grade_question(who_graded, grade_id, message, question, score)
+    comment = InlineComment.find_or_initialize_by(submission: self,  grade_id: grade_id, line: question)
+    comment.update(label: "Graded question",
+                   filename: self.assignment.name,
+                   severity: InlineComment.severities["info"],
+                   user_id: (who_graded || self.user).id,
```

Comment:
???  This is a change in semantics -- and on the face of it, it allows a student to comment on their own submission.  That doesn't seem right...